synapse/replication/tcp/redis.py

```diff
@@ -42,6 +43,24 @@
 
 logger = logging.getLogger(__name__)
 
+T = TypeVar("T")
+V = TypeVar("V")
+
+
+@attr.s(slots=True, frozen=True)
+class ConstantProperty(Generic[T, V]):
```

Comment:
Can you just use the `@property` decorator (or does that raise when attempting to set it?)

Reply:
Yeah, that raises. We could do `continueTrying = property(lambda _: True, lambda _a, _b: True)` but eww